### PR TITLE
add model opencsg-deepseek-coder-1.3b-v0.1

### PR DESCRIPTION
**Pull request by SeanHH86**
Support model opencsg-deepseek-coder-1.3b-v0.1

### CODE REVIEW

File: models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    look_back_period_s: 30.0
+    smoothing_factor: 1.0
+    downscale_delay_s: 300.0
+    upscale_delay_s: 90.0
+  ray_actor_options:
+    num_cpus: 0.5 # for a model deployment, we have 3 actor created, 1 and 2 will cost 0.1 cpu, and the model infrenece will cost 6(see the setting in the end of the file)
+model_config:
+  warmup: False
+  model_task: text-generation
+  model_id: OpenCSG/opencsg-deepseek-coder-1.3b-v0.1
```
Comment (depenglee1707) [inline models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml:17]: Which repository the `mode_id` refer to?  huggingface or csghub

File: models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  max_input_words: 800
+  initialization:
+    s3_mirror_config:
```
Comment (depenglee1707) [inline models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml:20]: Could you comment of remove this section, we expect model download runtime, so this section confuse people

File: models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml (hunk 1 above)
Comment (depenglee1707) [reply models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml:17]: BTW, why we need to support both `defaulttransformers` and `default `? is the behavior different?

**Review by depenglee1707 — CHANGES_REQUESTED** [review]
Thanks, please take a look my comments.

File: models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml (hunk 1 above)
Comment (SeanHH86) [reply models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml:17]: Right, just default is ok.

File: models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml (hunk 2 above)
Comment (SeanHH86) [reply models/text-generation--opencsg--opencsg-deepseek-coder-1.3b-v0.1-pipeline.yaml:20]: Comment those codes.

**Review by depenglee1707 — APPROVED** [review]
Great, thanks

**Pull request merged**